.travis.yml

```diff
@@ -59,4 +59,4 @@ matrix:
             - *required_packages
             - *optional_packages
 
-script: cmake -DCMAKE_BUILD_TYPE=RelWithDebInfo -DSUPPORT_JOURNALD=ON -DSUPPORT_DTLS="$DTLS" -DSUPPORT_ZMQ="$ZMQ" . && make && make test
+script: cmake -DCMAKE_BUILD_TYPE=RelWithDebInfo -DSUPPORT_JOURNALD=ON -DDISABLE_UDP_CONNECT=ON -DSUPPORT_DTLS="$DTLS" -DSUPPORT_ZMQ="$ZMQ" . && make && make test
```

Comment:
Did you add this to the testing config in order to not omit that code branch oder because you think this should be the default?

Reply:
I'm on the fence as to which should be default. Functional PMTUD is pretty common these days, but in our specific use case we disable the use of the connect.
I'll remove this change for the moment.